packages/app/src/themes/componentOverrides.ts

```diff
@@ -10,8 +11,50 @@ const redhatFont = `@font-face {
     url(/fonts/RedHatText-Regular.ttf) format('truetype');
 }`;
 
-export const components: UnifiedThemeOptions['components'] = {
-  MuiCssBaseline: {
-    styleOverrides: redhatFont,
-  },
+export const components = (
+  themeColors: ThemeColors,
+): UnifiedThemeOptions['components'] => {
+  return {
+    BackstageHeaderTabs: {
+      styleOverrides: {
+        tabsWrapper: {
+          paddingLeft: '0px',
+        },
+      },
+    },
+    MuiTabs: {
+      defaultProps: {
+        TabIndicatorProps: {
+          style: {
+            height: '3px',
+            background: themeColors.navigationIndicatorColor || '#0066CC',
+          },
+        },
+      },
+      styleOverrides: {
+        root: {
+          borderBottom: '1px solid #d2d2d2',
+        },
+      },
+    },
+    MuiTab: {
+      defaultProps: {
+        disableRipple: true,
+      },
+      styleOverrides: {
+        root: {
+          minWidth: 'auto',
+          '&:hover': {
+            boxShadow: '0 -3px #b8bbbe inset',
+          },
```

Comment:
I would also recommend to disable the text transformation that tab content is always UPPERCASE:

```suggestion
          // @ts-ignore: Tab contains a span 'wrapper'
          wrapper: {
            textTransform: 'none',
          },
          '&:hover': {
            boxShadow: '0 -3px #b8bbbe inset',
          },
```

Reply:
@jerolimov  Tried changing this by overriding the BackstageHeaderTabs textTransform, but the text looks really small

![Screenshot 2024-02-17 at 1 15 40 AM](https://github.com/janus-idp/backstage-showcase/assets/22490998/1e4563e1-1435-415e-a464-657d7701d909)
 
 cc @ShiranHi

Reply:
> @jerolimov Tried changing this by overriding the BackstageHeaderTabs textTransform, but the text looks really small

Can we change the text size so it will be like PF?
https://www.patternfly.org/components/tabs

Reply:
Done

![Screenshot 2024-02-19 at 11 35 23 AM](https://github.com/janus-idp/backstage-showcase/assets/22490998/ab855b71-064a-479b-bdc3-d30c97ff07f6)